Tasks/Common/webdeployment-common/webconfigutil.ts

```diff
@@ -77,14 +80,17 @@ export function addWebConfigFile(folderPath: any, webConfigParameters, rootDirec
             delete webConfigParameters['appType'];
 
             var selectedAppTypeParams = addMissingParametersValue(appType, webConfigParameters);
-            if(appType != "node") {
+            if(appType.startsWith("python")) {
                 rootDirectoryPath = "D:\\home\\" + (rootDirectoryPath ? rootDirectoryPath : "site\\wwwroot");
                 tl.debug('Root Directory path to be set on web.config: ' + rootDirectoryPath);
                 selectedAppTypeParams['KUDU_WORKING_DIRECTORY'] = rootDirectoryPath;
                 if(appType === 'python_Django' && webConfigParameters['DJANGO_SETTINGS_MODULE'].value === '') {
                     tl.debug('Auto detecting settings.py to set DJANGO_SETTINGS_MODULE...');
                     selectedAppTypeParams['DJANGO_SETTINGS_MODULE'] = getDjangoSettingsFile(folderPath);
                 }
+            } else if(appType == 'Go') {
+                rootDirectoryPath = "D:\\home\\" + (rootDirectoryPath ? rootDirectoryPath : "site\\wwwroot");
+                selectedAppTypeParams['GoExeFilePath'] = rootDirectoryPath + "\\" + webConfigParameters['GoExeFileName'].value;
```

Comment:
should we throw error, when the GoExeFileName value is empty/ null?

Reply:
`if(!!(webConfigParameters['GoExeFileName'].value)` helps in validating null and empty value.